flow360/plugins/report/report_items.py

```diff
@@ -106,6 +106,23 @@ class Settings(Flow360BaseModel):
     dpi: Optional[pd.PositiveInt] = 300
 
 
+class TableSettings(Settings):
+    """
+    Settings for controlling output tables.
+
+    Attributes
+    ----------
+    case_include : List[pd.NonNegativeInt], optional
+        The indices of cases to be included in the table
+
+    case_excluded : List[pd.NonNegativeInt], optional
+        The indices of cases to be excluded from the table
+    """
+
+    case_include: Optional[List[pd.NonNegativeInt]] = None
+    case_exclude: Optional[List[pd.NonNegativeInt]] = None
+
+
```

Comment:
in other charts we use "select_indices" and this here has very similar function. I would suggest to unify:
https://github.com/flexcompute/Flow360/blob/c721753c36ae0643e95014add61ae62bcad67b57/flow360/plugins/report/report_items.py#L413

I think `select_indices` is not the best name.